src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ClockConfig.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config
```

Comment:
Had to add a configurable clock so that certain tests can be run in on different dates